### ISSUE TITLE
Update Berts mods

### ISSUE THREAD

**Issue by pStyl3** [loot/skyrim#260]
[Berts Breezehome Cellar](https://www.nexusmods.com/skyrim/mods/15801)
[Berts Breezehome Remodel](https://www.nexusmods.com/skyrim/mods/8922)

Skyrim's masterlist currently contains the following plugins by `berticus0001`:
```
BertsBreezeCellar(esp.V.1.0).esp
BertsBreezeCellar(esp.V3.0).esp
BertsBreezehomeCellar.I.1.3.esp
BertsBreezehomeCellar(Grotto).esp
BertsBreezehomeGreenhouse.esp
BertsBreezehomeRemodel.I,.1.2.esp
BertsBreezehomeRemodel.I,.1.3.esp
BertsBreezehomeRemodel.I.1.4.esp
BertsBreezehomeRemodel.I.1.5.1.esp
BertsBreezehomeRemodel.3(Exterior).esp
0001BertsBreezehomeRemodel.V.2.esp
0001BertsBreezehomeRemodel.V.2(esp.2.0).esp
0001BertsBreezehomeRemodel.V.2(Grotto).esp
0001BertsBreezehomeRemodel.V.2(LydiaAddon.V.1).esp
```

From these plugins, the following 5 plugins ..
```
BertsBreezehomeCellar.I.1.3.esp
BertsBreezehomeRemodel.I,.1.2.esp
BertsBreezehomeRemodel.I,.1.3.esp
BertsBreezehomeRemodel.I.1.4.esp
BertsBreezehomeRemodel.I.1.5.1.esp
```
have messages attached to them in the form of:
```
 - name: 'BertsBreezehomeCellar.I.1.3.esp'
    msg:
      - type: warn
        content:
          - lang: en
            text: 'TES5Edit Msg - The volume for a file has been externally altered so that the opened file is no longer valid (BertsBreezehomeCellar.I.1.3.esp)'
```

The xEdit message seems suspicious. @CptMcSplody was able to find this commit that already has the data in it: https://github.com/loot/skyrim/commit/940260F4
CC: @egocarib 

So it was added at the `24 Jan 2014`, so at the time where BOSS transitioned to LOOT - or prior, then directly to BOSS.

In any case, I've asked @ElminsterAU if that message came from xEdit and what it could mean (in particular, the word `volume` in that context). To quote him:
```
I'm pretty sure xEdit never generated that entry as such... and that message is actually a file system error message that has absolutely nothing to do with xEdit
```
```
specifically, there was a file handle for that file, and then a process with system access opened a handle for the whole file system volume (giving direct read/write access to the whole partition, bypassing the file system) and made changes
0xC0000098     0x000003ee     NT_STATUS_FILE_INVALID     “The volume for a file has been externally altered such that the opened file is no longer valid.”
```
I first misunderstood the situation:
```
So practically this boils down to "do not use this file". I think I'm gonna change those messages in the masterlist to something more generic & less confusing.
```
But Elminster corrected me:
```
no, this boils down to: whoever added that message had messed up their computer and it has absolutely nothing to do with the file itself
```

So in conclusion:
- Add `location` info for all featured plugins
- Add new `clean` & `dirty` info using QAC from `xEdit 4.0.2f`
- Remove the above mentioned file system error messages
- Think about adding `obsolete` messages for all old plugins from Bert's mods

Also CC: @Sharlikran, you mentioned being interested in this.

**Comment by pStyl3** [comment]
Resolved through commit https://github.com/loot/skyrim/commit/40dfe3d8e857004b1455b5a98e52ab1138e1f23b & closing as such.
